packages/suite-desktop/src-electron/electron.ts

```diff
@@ -300,11 +292,14 @@ const init = async () => {
 app.name = APP_NAME; // overrides @trezor/suite-desktop app name in menu
 app.on('ready', init);
 
-// Quit when all windows are closed.
 app.on('window-all-closed', () => {
-    app.quit();
-    // @ts-ignore
-    mainWindow = undefined;
+    // On macOS it is common for applications and their menu bar
+    // to stay active until the user quits explicitly with Cmd + Q
+    if (process.platform !== 'darwin') {
+        app.quit();
+    } else {
+        mainWindow = null;
+    }
```

Comment:
Are we sure we want to keep this? It would be so much cleaner to just `app.quit()` for macOS as well (closing the bridge and tor we started).

Reply:
From what I observed on other applications and my understanding of the Electron docs in regards to Mac, the expected behaviour is to destroy the window but keep the app running in the tray. I don't think it makes any sense but I'm not a Mac user :shrug:

Reply:
Ah, OK, I get it now. Let's keep this then :-/